src/vivarium_csu_alzheimers/constants/data_keys.py

```diff
@@ -30,41 +30,43 @@ def log_name(self):
 POPULATION = __Population()
 
 
-# TODO - sample key group used to identify keys in model
-# For more information see the tutorial:
-# https://vivarium-inputs.readthedocs.io/en/latest/tutorials/pulling_data.html#entity-measure-data
-class __SomeDisease(NamedTuple):
-
+class __Alzheimers(NamedTuple):
     # Keys that will be loaded into the artifact. must have a colon type declaration
-    SOME_DISEASE_PREVALENCE: TargetString = TargetString("cause.some_disease.prevalence")
-    SOME_DISEASE_INCIDENCE_RATE: TargetString = TargetString(
-        "cause.some_disease.incidence_rate"
+    PREVALENCE: TargetString = TargetString(
+        "cause.alzheimers_disease_and_other_dementias.prevalence"
     )
-    SOME_DISEASE_REMISSION_RATE: TargetString = TargetString(
-        "cause.some_disease.remission_rate"
+    INCIDENCE_RATE: TargetString = TargetString(
+        "cause.alzheimers_disease_and_other_dementias.incidence_rate"
+    )
+    REMISSION_RATE: TargetString = TargetString(
+        "cause.alzheimers_disease_and_other_dementias.remission_rate"
+    )
+    DISABILITY_WEIGHT: TargetString = TargetString(
+        "cause.alzheimers_disease_and_other_dementias.disability_weight"
+    )
+    EMR: TargetString = TargetString(
+        "cause.alzheimers_disease_and_other_dementias.excess_mortality_rate"
+    )
+    CSMR: TargetString = TargetString(
+        "cause.alzheimers_disease_and_other_dementias.cause_specific_mortality_rate"
```

Comment:
does this get used somehow?

Reply:
I think this won't get used until we implement the intervention components.

Reply:
Actually I think this does get used in the mortality components right now but somewhere deep inside VPH.